src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $cards-width;
+  background: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &:last-child {
+    margin-right: 0;
+  }
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__product-name {
+    color: var(--primary-grey);
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
```

Comment:
you can try to use `mixins` to create some typography styles, put there color, font-size, line-height, font-weight